lib/tasks/csv_report.rake

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "csv"
+
+namespace :csv_report do
+  desc "A CSV report of political documents published by organisations"
+  task political_status_by_organisation: :environment do
+    raise "csv_report:political_status_by_organisation rake task should be removed" unless Time.zone.today < Date.new(2020, 1, 1)
```

Comment:
This behaviour isn't captured in a test nor in the Trello card. What are its origins?

Reply:
I'll add a bit of backstory...

It's very common in our rake tasks (and even some app features) that are used once or twice then never used again but remain in the code for perpetuity. If we can forsee that being the case it's worth adding the equivalent of dynamite so that when we hit a certain point in time we have to deal with either removing the feature, extending the date or removing the dynamite.

This feature seems something that's unlikely to be needed long term once we have filtering in so seems a candidate for removal.

Though I would suggest it be flagged in it's own commit so the intent is documented.

Reply:
Kevin suggested this approach. I will try to remove the task before it causes any annoyance.